netci.groovy

```diff
@@ -1095,10 +1108,17 @@ combinedScenarios.each { scenario ->
                                 case 'x64':
                                 case 'x86ryujit':
                                 case 'x86lb':
+                                    def arch = ''
+                                    if (architecture == 'x64') {
+                                        arch = architecture
+                                    }
+                                    else {
+                                        arch = 'x86'
+                                    }
                                     
```

Comment:
How about:
def arch = architecture
if (architecture == 'x86ryujit' || architecture == 'x86lb) {
   arch = 'x86'
}

That way, it's correct as other architectures are added.